gcsweb.k8s.io/README.md

```diff
@@ -32,3 +32,28 @@ https://github.com/kubernetes/test-infra/tree/master/gcsweb into Docker
 container, which is then uploaded to private container storage at
 https://k8s.gcr.io/gcsweb-amd64 and fetched during processing
 of `deployment.yaml` by `kubectl apply`.
+
+##### How to deploy
+
+Ensure you have [access to the cluster]
+
+Ensure you are a member of both:
+
+- k8s-infra-cluster-admins@kubernetes.io
```

Comment:
You should only need to be in `k8s-infra-rbac-gcsweb@kubernetes.io` to work within that single namespace.  If that is not sufficient, something is wrong.

`k8s-infra-cluster-admins` is basically root.

Reply:
If deployment involves updating secrets then this is needed. If not, it isn't

I want us to revisit that constraint at some point, to allow namespace owners to admin their own secrets if possible

Reply:
Yeah, happy to revisit.  The main thinking was to defend .k8s.io certs.

I think we should NOT document this as a group leaf app-operators should join.

Reply:
I'll update the docs with a following PR.